environment.yml

```diff
@@ -13,12 +13,11 @@ dependencies:
   - numpy<2.0.0
   - openfe-analysis>=0.3.1
   - openff-interchange
-  - openff-models>=0.0.5
   - openff-nagl-base >=0.3.3
   - openff-nagl-models>=0.1.2
   - openff-toolkit>=0.16.2
   - openff-units==0.3.1
-  - openmm >=8.2.0
+  - openmm !=8.1.0, <8.3.0
```

Comment:
@mikemhenry is there any reason we'd want a min of 8.2 here? (i know we don't want the 8.2 min on the feedstock)

Reply:
We probably want a min of 8.1.1, there's stuff we use for salvation that needs it.

Reply:
okay, how about `openmm ~=8.2.0` so we're testing against the lasted qualified version, and also catching any patches that come in?